custom_components/tuya_local/devices/pir_spotlight.yaml

```diff
@@ -0,0 +1,184 @@
+name: PIR Spotlight
+products:
+  - id: reldobj3ny6fbgvw
+    name: WiFi Duo LED Light WFD 3050 P
+
+primary_entity:
+  entity: light
+  dps:
+    - id: 20
+      type: boolean
+      name: switch
+    - id: 21
+      type: string
+      name: work_mode
+      mapping:
+        - dps_val: colour
+          value: hs
+        - dps_val: white
+          value: white
+        - dps_val: music
+          value: Music
+        - dps_val: scene
+          value: Scene
+    - id: 22
+      name: brightness
+      type: integer
+      range:
+        min: 10
+        max: 1000
+      mapping:
+        - dps_val: null
+        - scale: 3.92
+    - id: 23
+      name: color_temp
+      type: integer
+      optional: true
+      range:
+        min: 0
+        max: 1000
+      mapping:
+        - invert: true
+secondary_entities:
+  - entity: number
+    name: Timer
+    category: config
+    icon: "mdi:timer"
+    dps:
+      - id: 26
+        name: value
+        type: integer
+        optional: true
+        range:
+          min: 0
+          max: 86400
+        unit: min
+        mapping:
+          - scale: 60
+            step: 60
+          - dps_val: null
+  - entity: switch
+    name: Manual Mode
+    category: config
+    dps:
+      - id: 51
+        type: string
+        name: switch
+        mapping:
+        - dps_val: auto
+          value: false
+        - dps_val: manual
+          value: true
+        optional: true
+        force: true

```

Comment:
I don't know if `force: true` will work here (and other locations).  Having read through some threads on localtuya and tinytuya, the method used to force dps to report seems to cause problems for values that can be set, and the only positive reports of it making a difference is for Voltage/Current/Power readings from smartplugs.